File: brian_ortega_delahoz/Clases/Programación/Mayo2024/31-May-24/notas.rb
puts "Ingrese la cantidad de estudiantes: "
n = gets.chomp.to_i

for cantEst in 1..n
    puts "Ingrese la nota 1: "
    not1 = gets.chomp.to_f
  
    puts "Ingrese la nota 2: "
    not2 = gets.chomp.to_f
  
    puts "Ingrese la nota 3: "
    not3 = gets.chomp.to_f

    prom = (not1 + not2 + not3) / 3
    
    if prom >= 6
        puts "Felicidades, Aprobaste!!"
    else
        puts "Reprobaste la materia :("
    end
end

File: brian_ortega_delahoz/Clases/Programación/Mayo2024/31-May-24/notas.dart
import 'dart:io';

void main(){
 print("Ingresar la cantidad de estudiantes:");
  int a = int.parse(stdin.readLineSync()!);
  for (int i = 1; i <= a; i++) {
    print("Ingresa tu primera nota:");
    double n1 = double.parse(stdin.readLineSync()!);

    print("Ingresa tu segunda nota:");
    double n2 = double.parse(stdin.readLineSync()!);

    print("Ingresa tu tercera nota:");
    double n3 = double.parse(stdin.readLineSync()!);

    double promedio = (n1+n2+n3)/3;

    if(promedio>=6){
      print("Felicidades, Aprobaste!!");

    }else{
      print("Reprobaste :(");
    }


}
 }
